packages/utils/src/usePaginate/usePaginate.js

```diff
@@ -0,0 +1,48 @@
+import { useState } from 'react';
+
+/**
+ * Provides `pagination` props and functionality for a (Primary)Toolbar
+ *
+ * @param {Object} [options]
+ */
+const usePaginate = (options = {}) => {
```

Comment:
Perhaps `usePagination` is a bit more clear naming

Reply:
Renamed.